health/notifications/web/README.md

```diff
@@ -1,8 +1,23 @@
 # Dashboard
 
+
+## Notification
+
 The netdata dashboard shows HTML notifications, when it is open.
 
 Such web notifications look like this:
 ![image](https://cloud.githubusercontent.com/assets/2662304/18407279/82bac6a6-7714-11e6-847e-c2e84eeacbfb.png)
 
+## Alarms
+
+The netdata GUI has a link for the alarms raised in the system on top menu, when the user access it, (s)he will have the following options:
+
+- Active : The active alarms of the Netdata. This option does not include the alarms that were not raised.
```

Comment:
`The active alarms of the Netdata` you should remove the `the `.

Reply:
Thanks, I corrected the grammar.